### ISSUE TITLE
WIKI addition. 

### ISSUE THREAD

**Issue by rewolff** [mcallegari/qlcplus#1020]
I would gladly edit the wiki myself, but can't find an edit link... But looking for build instructions on linux, I found "LInux build HOWTO". 

So I happily followed the instructions and in the end got something that didn't do what I wanted it to do because apparently that feature depends on QT5. 

After several attempts myself of trying to force QT5 complication I finally found the QT5 instuctions. 

IMHO there should be a link in the "normal" build instructions that says something like: 

**This howto here builds using the QT4 library. (which _is / is not_ (*) recommended) If you wish to build with QT5, please see this page <https://github.com/mcallegari/qlcplus/wiki/Linux-build-Qt5>** 

(*) You chose.

**Comment by mcallegari** [comment]
Huh ?
1- go to QLC+ website homepage ([http://www.qlcplus.org/](http://www.qlcplus.org/))
2- scroll until, on the right, you see "QLC+ Build HOWTO"
3- Pick between Linux Qt4 / Linux Qt5 depending on your distribution

I think it's clear enough to find.

"not recommended" is just your opinion. There might be users on old distribution that didn't include Qt5. So Qt4 builds are still perfectly valid, even though they don't include video playback for example.

**Comment by rewolff** [comment]
**"not recommended" is just your opinion.** 
No... not my opinion. I just provided a convenient place for YOU to express YOUR opinion on the matter. If you don't care either, then just leaving that remark out is fine as well. 

YOU have designed a "neat" way to navigate to the page I was looking for. When you follow that route, you are right and it is quite obvious I can make the right choice there. But somehow, I did not arrive on that page through your suggested route. Maybe through google. 

So then maybe adding: "These are the instructions for building on Linux with QT4. If you want to build for windows, macos, or with QT5, see https://github.com/mcallegari/qlcplus/wiki " would have helped me.
